packages/cubejs-query-orchestrator/src/orchestrator/IORedisFactory.ts

```diff
@@ -1,10 +1,14 @@
 import Redis, { Redis as redis, RedisOptions } from 'ioredis';
-import { getEnv } from '@cubejs-backend/shared';
+import { getEnv, devLogger, prodLogger } from '@cubejs-backend/shared';
 import AsyncRedisClient from './AsyncRedisClient';
 import { parseRedisUrl } from './utils';
 
 export type IORedisOptions = RedisOptions;
 
+const logger = process.env.NODE_ENV !== 'production'
```

Comment:
The logger should be re-used from the `server-core`, instead of creating a new one.